src/main/java/org/jitsi/srtp/SrtpCryptoContext.java

```diff
@@ -505,6 +505,14 @@ synchronized public SrtpErrorStatus reverseTransformPacket(ByteArrayBuffer pkt,
         long guessedIndex = guessIndex(seqNo);
         SrtpErrorStatus ret, err;
 
+        if (policy.getEncType() == SrtpPolicy.AESGCM_ENCRYPTION) {
+            /* We can't skip decryption for GCM, because it's also the authentication. */
+            /*  (Note: in theory it'd be possible to run a GCM auth without the decryption
+             *   part, but I don't know of any GCM cipher APIs which support this.)
+             */
+            skipDecryption = false;
```

Comment:
I wonder how much of a perf hit this will be in practice (given that muted participants keep sending silent packets ~ 20kbps that the bridge discards) and whether that's negligible compared to the GCP perf gains.

Reply:
I'm experimenting with a solution for this.

Reply:
Alternately, there's Boris's fix where we just entirely discard silent packets (don't even authenticate them).

Reply:
Discarding silent packets entirely will break non-JVB setups, e.g. Jigasi with Asterisk or any other encrypted SIP/XMPP session.

Reply:
We've been discarding ingress silence packets for a while now. Sequence number gaps are masked on the egress of course. Do you see any other issues?

Reply:
No, it's only the sequence numbers. But I wonder if the same trick works in the legacy code (libjitsi, ie. Desktop and thus Jigasi).

Reply:
I think both Desktop and Jigasi can benefit from the trick, as long as the audio packets come with the RTP audio level header extension: both the ssrc transform engine (that flags packets for discard) and the discard transform engine (that masks the sequence number gaps) are still included in the media stream transform chain.

Reply:
Ok. You definitely know these parts better than I do.

Reply:
This perf hit is fixed by https://github.com/jitsi/jitsi-srtp/pull/23